spec/features/users/user_can_get_data_from_github_spec.rb

```diff
@@ -21,5 +24,8 @@
 
     expect(page).to have_content("Followers")
     expect(page).to have_link("leiyakenney")
+
+    expect(page).to have_content("Following")
+    expect(page).to have_link("hillstew")
```

Comment:
Looks good! The only thing we might need to discuss is the hardcoded repo name, follower name, and following name. I played around with interpolating something instead, but that doesn't work from what I've tried. The main reason I think it might be a problem is because I am not following "hillstew" on my github.

Reply:
As you mentioned during our discussion, the information is retrieved from our Fixtures files :)